Document `@encodedName` vs `@clientName` (#424)
Azure/typespec-azure: docs/howtos/DataPlane Generation - DPG/04renaming.mdx

This page documents how to customize names for client generations in DPG. For an overview of the setup, please visit the setup page.

:::note

> The TypeSpec compiler provide an `@encodedName` decorator that allows changing the name of the property for a given serialization format.

Reviewer: ```suggestion
The TypeSpec compiler provides an `@encodedName` decorator that allows changing the name of the property for a given serialization format.
```

> However in Azure we recommend that you define the property name as the wire name an use the `@clientName` decorator to change the name in the client instead.

Reviewer 2: ```suggestion
However in Azure we recommend that you define the property name as the value sent on the wire and use the `@clientName` decorator to change the name of the generated property.
```
